csi/server/plugin/opensds/controller.go

```diff
@@ -8,12 +8,56 @@ import (
 	sdscontroller "github.com/opensds/nbp/client/opensds"
 	"github.com/opensds/opensds/pkg/model"
 	"golang.org/x/net/context"
+	"fmt"
+	client2 "github.com/opensds/opensds/client"

```

Comment:
the `sdscontroller "github.com/opensds/nbp/client/opensds"` has encapsulated the `github.com/opensds/opensds/client`, so it's best to use only `sdscontroller` here